shared/team-building/index.tsx

```diff
@@ -295,6 +296,10 @@ class TeamBuilding extends React.PureComponent<Props> {
     this.props.fetchUserRecs()
   }
 
+  componentDidUpdate = () => {
+    this.props.refreshBlockList?.()
```

Comment:
After reading more of relevant code, I'm not sure this is the proper place to have this. It seems you could have something like this in `actions/users.tsx`:

```
yield* Saga.chainAction2(TeamBuildingGen.searchResultsLoaded, refreshBlockList)
```

Reply:
K, will try that.

Reply:
thanks!